.github/workflows/pr-gate.yml

```diff
@@ -93,6 +93,7 @@ jobs:
     - name: Run ${{ matrix.test-kind }} tests ('${{ matrix.test-marker }}')
       uses: ./.github/workflows/actions/run-tests
       with:
+        tox-env: 'cpu' # run the cpu tests with the 'recommenders[dev,examples]' dependencies
```

Comment:
@laserprec, this is really good! These comments helps a new reader to understand what is happening very quickly